openpype/modules/kitsu/actions/launcher_show_in_kitsu.py

```diff
@@ -0,0 +1,112 @@
+import webbrowser
+
+from openpype.pipeline import LauncherAction
+from openpype.modules import ModulesManager
+from openpype.client import get_project, get_asset_by_name
+
+
+class ShowInKitsu(LauncherAction):
+    name = "showinkitsu"
+    label = "Show in Kitsu"
+    icon = "external-link-square"
+    color = "#e0e1e1"
+    order = 10
+
+    @staticmethod
+    def get_kitsu_module():
+        return ModulesManager().modules_by_name.get("kitsu")
+
+    def is_compatible(self, session):
+        if not session.get("AVALON_PROJECT"):
+            return False
+
+        return True
+
+    def process(self, session, **kwargs):
+
+        # Context inputs
+        project_name = session["AVALON_PROJECT"]
+        asset_name = session.get("AVALON_ASSET", None)
+        task_name = session.get("AVALON_TASK", None)
+
+        project = get_project(project_name=project_name,
+                              fields=["data.zou_id"])
+        if not project:
+            raise RuntimeError(f"Project {project_name} not found.")
+
+        project_zou_id = project["data"].get("zou_id")
+        if not project_zou_id:
+            raise RuntimeError(f"Project {project_name} has no "
+                               f"connected ftrack id.")
+
+        asset_zou_data = None
+        task_zou_id = None
+        if asset_name:
+            asset_fields = ["data.zou.id", "data.zou.type"]
+            if task_name:
+                asset_fields.append(f"data.tasks.{task_name}.zou.id")
+
+            asset = get_asset_by_name(project_name,
+                                      asset_name=asset_name,
+                                      fields=asset_fields)
+
+            asset_zou_data = asset["data"].get("zou")
+            if not asset_zou_data:
+                raise RuntimeError(
+                    f"No zou asset data for asset: {asset_name}"
+                )
+
+            if task_name:
+                task_data = asset["tasks"][task_name]
```

Comment:
```suggestion
                task_data = asset["data"]["tasks"][task_name]
```
There is an error with the "tasks" key of the list, you must add the "data" key before.

Reply:
Ah sorry, I missed that you had a "suggested" change.